### DISCUSSION TITLE
Security problem (allow-scripts and allow-same-origin)

### DISCUSSION THREAD

**Discussion by nikitakoliadin** [keycloak/keycloak#25343] [Q&A]
<img width="807" alt="Screenshot 2023-12-06 at 18 22 53" src="https://github.com/keycloak/keycloak/assets/35656436/606871a3-f885-4961-bbe3-66efd031a3c4">

I have default Security defenses settings and web origin is set to '+'.

keycloak and keycloak-js are both 23.0.1.

What I do wrong and how to fix the issue ?

**Comment by nikitakoliadin** [comment]
Reproduced in Chrome.

**Comment by lazyest** [comment]
same in FF

**Reply by codespearhead** [reply]
What page throws this error?

Is that problem reproducible in Keycloak v24?

If so, tell us the steps to reproduce the issue on our ends.

**Comment by lazyest** [comment]
In my case it was latest docker image run, after creating new user I've tried to login with new creds and got infinite spinning wheel instead. In browser console I saw mentioned error.

**Comment by Nosfistis** [comment]
This is still happening in Keycloak v25.0.2

**Comment by JuamBer** [comment]
Still happening with in local/prod environments:

> "keycloak-angular": "^19.0.2",
> "keycloak-js": "^26.1.1",

<img width="1915" height="477" alt="screenshot" src="https://github.com/user-attachments/assets/1e1d916a-46a3-489c-83f7-dcf3fa9e1011" />


**silent-check-sso.html**
```
<html>
  <body>
    <script>
      parent.postMessage(location.href, location.origin);
    </script>
  </body>
</html>
```
**app.config.ts**
```
provideKeycloak({
      config: environment.keycloak_config,
      initOptions: {
        onLoad: 'check-sso',
        silentCheckSsoRedirectUri: window.location.origin + '/assets/silent-check-sso.html',
      },
      providers: [
        {
          provide: INCLUDE_BEARER_TOKEN_INTERCEPTOR_CONFIG,
          useValue: [
            createInterceptorCondition<IncludeBearerTokenCondition>({
              urlPattern: /.*/,
            }),
          ],
        },
      ],
      features: [],
}),
```

**Comment by mishurov** [comment]
Still happening
```json
"next": "15.5.4",
"keycloak-js": "26.2.1",
```
